packages/app/src/TeamPage.tsx

```diff
@@ -31,6 +31,13 @@ export default function TeamPage() {
     rotateApiKeyConfirmationModalShow,
     setRotateApiKeyConfirmationModalShow,
   ] = useState(false);
+  const [
+    deleteTeamMemberConfirmationModalData,
+    setDeleteTeamMemberConfirmationModalData,
+  ] = useState({
```

Comment:
mega nit: i feel like this could be simplified by using a nullable string instead of needing to encode both show and email separately.

Reply:
Thanks for the review, changes made